cpp/src/tsvd/tsvd.cuh

```diff
@@ -133,25 +131,17 @@ void calEig(const raft::handle_t& handle,
  * @param n_cols: number of columns of input matrix
  * @param components: components matrix.
  * @param n_cols_comp: number of columns of components matrix
- * @param allocator device custom allocator object
  * @param stream cuda stream
  * @{
  */
 template <typename math_t>
-void signFlip(math_t* input,
-              int n_rows,
-              int n_cols,
-              math_t* components,
-              int n_cols_comp,
-              std::shared_ptr<raft::mr::device::allocator> allocator,
-              cudaStream_t stream)
+void signFlip(
+  math_t* input, int n_rows, int n_cols, math_t* components, int n_cols_comp, cudaStream_t stream)
 {
   auto counting = thrust::make_counting_iterator(0);
   auto m        = n_rows;
 
-  ML::thrustAllocatorAdapter alloc(allocator, stream);
-  auto execution_policy = thrust::cuda::par(alloc).on(stream);
-  thrust::for_each(execution_policy, counting, counting + n_cols, [=] __device__(int idx) {
+  thrust::for_each(rmm::exec_policy(stream), counting, counting + n_cols, [=] __device__(int idx) {
```

Comment:
`handle.get_thrust_policy`

Reply:
Unless I missed something, this function shouldn't have access to a handle.